pages/index.js

```diff
@@ -1,69 +1,93 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import { useState, useEffect } from 'react';
+import { Stack, Button } from '@mui/material';
+import saveAs from 'file-saver';
+import SelectMenu from '../components/SelectMenu';
```

Comment:
try to structure your imports. include a line space between 3rd party imports and local imports.